osu.Game/Overlays/Profile/Header/CentreHeaderContainer.cs

```diff
@@ -85,6 +87,13 @@ private void load(OverlayColourProvider colourProvider)
                     }
                 }
             };
+
+            User.BindValueChanged(user => updateDisplay(user.NewValue?.User));
```

Comment:
move to `LoadComplete()` and call as `BindValueChanged(..., true)` to avoid possible lack of first update